README.md

```diff
@@ -387,12 +409,18 @@ yarn format-package --help
 6.  [`format-package.config.yaml` or `format-package.config.yml`](#with-format-packageymlyaml-format-packageconfigymlyaml)
 7.  [`format-package`](#with-packagejson) property in `package.json`
 
-If there are no valid configuration, it will use the [default](lib/defaults/index.js).
+If there are no valid configuration from the above search places, move up one directory level and try again.
+So the search continues in ./, ../, ../../, ../../../, etc., checking the same places in each directory.
+
+`format-package` will continue searching until arriving at your home directory.
```

Comment:
A style choice I made when I wrote the `README` was to avoid the use of "you". I messed this up in two places that I need to fix. 

For this line, can we phrase this similar to:

>`format-package` will continue searching until arriving at the home directory.

Reply:
Done.